specification/peering/resource-manager/Microsoft.Peering/preview/2019-03-01-preview/peering.json

```diff
@@ -0,0 +1,1449 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "resource": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "operation": {
+          "description": "The name of the operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "description": {
+          "description": "The description of the operation.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PeerAsn": {
+      "description": "The essential information related to the pee's ASN.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PeerAsnProperties",
+          "description": "The properties that define a peer's ASN.",
+          "x-ms-client-flatten": true
+        },
+        "name": {
+          "description": "The name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "description": "The ID of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PeerAsnProperties": {
+      "description": "The properties that define a peer's ASN.",
+      "type": "object",
+      "properties": {
+        "peerAsn": {
+          "format": "int32",
+          "description": "The Autonomous System Number (ASN) of the peer.",
+          "type": "integer"
+        },
+        "peerContactInfo": {
+          "$ref": "#/definitions/ContactInfo",
+          "description": "The contact information of the peer."
+        },
+        "peerName": {
+          "description": "The name of the peer.",
+          "type": "string"
+        },
+        "validationState": {
+          "description": "The validation state of the ASN associated with the peer.",
+          "enum": [
+            "None",
+            "Pending",
+            "Approved",
+            "Failed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "validationState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ContactInfo": {
+      "description": "The contact information of the peer.",
+      "type": "object",
+      "properties": {
+        "emails": {
+          "description": "The list of email addresses.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "phone": {
+          "description": "The list of contact numbers.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PeerAsnListResult": {
+      "description": "The paginated list of [T].",
```

Comment:
should it have a more descriptive description?

Reply:
complete